packaging/suse/patches/0_change_versions.gem.patch

```diff
@@ -7,7 +7,7 @@ index 5a5b38d20e7e..309f537a8fa2 100644
 
  # If the deployment is done through Puma, include it in the bundle.
 -gem "puma", "~> 3.7.0" if ENV["PORTUS_PUMA_DEPLOYMENT"] == "yes" || !packaging?
-+gem "puma", "~> 2.16.0" if ENV["PORTUS_PUMA_DEPLOYMENT"] == "yes" || !packaging?
++gem "puma", "= 2.16.0" if ENV["PORTUS_PUMA_DEPLOYMENT"] == "yes" || !packaging?
```

Comment:
It comes from a rebase I think. I introduced this change in a commit that happened when this was being reviewed-developed.